internal/cmd/iam/command_rbac_role_binding.go

```diff
@@ -363,12 +361,12 @@ func (c *roleBindingCommand) displayCCloudCreateAndDeleteOutput(cmd *cobra.Comma
 	// The err is ignored here since the --prefix flag is not defined by the list subcommand
 	prefix, _ := cmd.Flags().GetBool("prefix")
 
-	resource, err := cmd.Flags().GetString("resource")
+	res, err := cmd.Flags().GetString("resource")
```

Comment:
```suggestion
	resource, err := cmd.Flags().GetString("resource")
```

Reply:
If I do that I need to import `internal/pkg/resource` with a different name, since that package is used in the same function that this change is in.